make/test.mk

```diff
@@ -49,8 +49,8 @@ test-e2e: ./vendor e2e-setup
 ifeq ($(OPENSHIFT_VERSION),3)
 	$(Q)oc login -u system:admin
 endif
-	$(Q)operator-sdk test local ./test/e2e --namespace $(TEST_NAMESPACE) --up-local --go-test-flags "-v -timeout=10m"
-	# $(Q)operator-sdk test local ./test/e2e --image quay.io/openshift-pipeline/openshift-pipelines-operator:v0.4.0-1 --namespace $(TEST_NAMESPACE)
+	# $(Q)operator-sdk test local ./test/e2e --namespace $(TEST_NAMESPACE) --up-local --go-test-flags "-v -timeout=10m"
+	$(Q)operator-sdk test local ./test/e2e --image quay.io/openshift-pipeline/openshift-pipelines-operator:v0.4.0-1 --namespace $(TEST_NAMESPACE)
```

Comment:
You probably want to make it configurable (the version number)

Reply:
watch this one
https://github.com/openshift/tektoncd-pipeline-operator/pull/55/commits/0524de4e01906b796e69f92a30141a94cce0dcd8
pr https://github.com/openshift/tektoncd-pipeline-operator/pull/55